RDoc-2600 Describe RavenDB Cloud Product Features (#1895)
ravendb/docs: Documentation/6.0/Raven.Documentation.Pages/cloud/cloud-features.markdown

> # Cloud: Product Features

Reviewer: I would add a note about the impact of enabling / disabling the features. IIRC in general these actions restart the RavenDB service - we should notify about that by adding a `WARNING` section in the particular feature descriptions. The same for editing the features settings (Power BI, Monitoring IPs etc.).

---

{PANEL: OLAP ETL }

> RavenDB Cloud provides OLAP ETL capabilities, allowing you to integrate your operational data with analytical data

Reviewer: ```suggestion
RavenDB Cloud provides **OLAP ETL** capabilities, allowing you to integrate your operational data with analytical data
```

warehouses.

Usage of this feature is described [here](../studio/database/tasks/ongoing-tasks/olap-etl-task).

{PANEL/}

{PANEL: PowerBi }

> RavenDB Cloud supports direct integration with Power BI, a leading business analytics tool. This feature allows you

Reviewer: ```suggestion
RavenDB Cloud supports direct integration with **Power BI**, a leading business analytics tool. This feature allows you
```

> to seamlessly connect your RavenDB data with Power BI for advanced visualization and reporting, enabling real-time

Reviewer: ```suggestion
to seamlessly connect your RavenDB data with **Power BI** for advanced visualization and reporting, enabling real-time
```

insights and data-driven decision-making.

Usage of this feature is described [here](../integrations/postgresql-protocol/power-bi).

## Configuration

{WARNING: Required }

> PowerBi IPs must be configured after enabling PowerBi feature.

Reviewer: ```suggestion
PowerBi IPs must be configured **after** enabling PowerBi feature.
```

{WARNING/}

> To access PowerBi Configuration View click **Configure** Button in PowerBi Feature Row. Here you can edit your IP

Reviewer: ```suggestion
To access *Power BI* Configuration View click **Configure** Button in *Power BI* feature row. Here you can edit your IP
```

> addresses that will have access to RavenDB PowerBi port.

Reviewer: ```suggestion
addresses that will have access to *RavenDB Power BI* port.
```

> !["PowerBi Configuration View"](images/product-features-powerbi-configuration.png "PowerBi Configuration View")

Reviewer: ```suggestion
!["Power BI Configuration View"](images/product-features-powerbi-configuration.png "Power BI Configuration View")
```

{INFO: }

> PowerBi IPs entries are [CIDR ranges](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing#CIDR_notation)

Reviewer: ```suggestion
*Power BI* IPs entries are [CIDR ranges](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing#CIDR_notation)
```

that define networks from which the connection is allowed.
{INFO/}

{PANEL/}

{PANEL: Queue ETL }

> The Queue ETL feature enables you to export data from RavenDB to various queue systems, such as RabbitMQ or Kafka.

Reviewer: ```suggestion
The *Queue ETL* feature enables you to export data from *RavenDB* to various queue systems, such as *RabbitMQ* or *Kafka*.
```

This is particularly useful for scenarios where you need to process data asynchronously or distribute it
to other microservices and applications.

Usage of this feature is described [here](../server/ongoing-tasks/etl/queue-etl/overview).

{PANEL/}

{PANEL: Queue Sink }

> Queue Sink allows RavenDB Cloud to directly receive data from queues, making it easier to ingest data into your

Reviewer: ```suggestion
*Queue Sink* allows *RavenDB Cloud* to directly receive data from queues, making it easier to ingest data into your
```

database from external systems or services. This feature is ideal for integrating with event-driven architectures
where data is processed and ingested in real time.

Usage of this feature is described [here](../server/ongoing-tasks/queue-sink/overview).

{INFO: Availability }

> This feature is available only for RavenDB 6.0 and newer.

Reviewer: ```suggestion
This feature is available only for *RavenDB* **6.0** and newer**.
```

{INFO/}

{PANEL/}

{PANEL: Data Archival }

> RavenDB Cloud offers the ability to archive selected documents.

Reviewer: ```suggestion
*RavenDB Cloud* offers the ability to archive selected documents.
```

> Archived documents are compressed and can be handled differently by RavenDB functions (e.g. Indexing can exclude

Reviewer: ```suggestion
Archived documents are compressed and can be handled differently by *RavenDB* functions (e.g. Indexing can exclude
```

archived documents from indexes and Data Subscriptions can avoid sending archived docs to workers), helping to keep the
database smaller and quicker and its contents more relevant.

Usage of this feature is described [here](../server/extensions/archival).

{INFO: Availability }

> This feature is available only for RavenDB 6.0 and newer.

Reviewer: This feature is available only for *RavenDB* **6.0** and newer**.

> This feature is available only for RavenDB 6.0 and newer.

Reviewer: ```suggestion
This feature is available only for *RavenDB* **6.0** and newer**.
```

{INFO/}

{PANEL/}

{PANEL: Monitoring }

> RavenDB Cloud allows you to monitor your RavenDB Cloud instances via SNMP protocol. SNMP is a widely used network

Reviewer: ```suggestion
*RavenDB Cloud* allows you to monitor your *RavenDB Cloud* instances via *SNMP* protocol. *SNMP* is a widely used network
```

management protocol that allows for the collection and organization of information about managed devices on IP networks.

## Configuration

> Click **Configure** Button in Monitoring Feature Row in order to configure this feature.

Reviewer: ```suggestion
Click **Configure** Button in Monitoring feature row in order to configure this feature.
```

You will see Monitoring Configuration View. You can edit SNMP Monitoring IPs in the first section.
The second section displays SNMPv3 Monitoring credentials.

!["Monitoring Configuration View"](images/product-features-monitoring-configuration.png "Monitoring Configuration View")

### SNMP Monitoring IPs

{WARNING: Required }

> SNMP Monitoring IPs must be configured after enabling Monitoring feature.

Reviewer: ```suggestion
*SNMP Monitoring* IPs must be configured **after** enabling *Monitoring* feature.
```

{WARNING/}

> This section allows you to specify IPs that will have access to your product using SNMP protocol.

Reviewer: ```suggestion
This section allows you to specify IPs that will have access to your product using *SNMP* protocol.
```

!["Edit SNMP Monitoring IPs"](images/product-features-monitoring-edit-ips.png "Edit SNMP Monitoring IPs")

### SNMPv3 Monitoring credentials

> SNMPv3 is the most secure version of SNMP, providing enhanced security features over its predecessors. When setting up

Reviewer: ```suggestion
*SNMPv3* is the most secure version of *SNMP*, providing enhanced security features over its predecessors. When setting up
```

> SNMPv3 monitoring for your RavenDB Cloud instances, you need to configure specific credentials that ensure secure

Reviewer: ```suggestion
SNMPv3 monitoring for your *RavenDB Cloud* instances, you need to configure specific credentials that ensure secure
```

> SNMPv3 monitoring for your RavenDB Cloud instances, you need to configure specific credentials that ensure secure

Reviewer: ```suggestion
*SNMPv3* monitoring for your *RavenDB Cloud* instances, you need to configure specific credentials that ensure secure
```

> communication between the SNMP agent on your end and the RavenDB.

Reviewer: ```suggestion
communication between the *SNMP* agent on your end and the *RavenDB*.
```

> Credentials required to connect to RavenDB Monitoring Endpoints are provided in this section. We also provide sample

Reviewer: ```suggestion
Credentials required to connect to *RavenDB Monitoring* Endpoints are provided in this section. We also provide sample
```

command you can use to test connection.

{INFO: }

> SNMP Monitoring IPs entries

Reviewer: ```suggestion
*SNMP Monitoring* IPs entries
```
